src/component/page-list/page-tile-container.tsx

```diff
@@ -0,0 +1,94 @@
+import * as MobX from 'mobx';
+import { observer } from 'mobx-react';
+import { PageRef } from '../../store/page/page-ref';
+import { PageTileComposite } from './page-tile-composite';
+import * as React from 'react';
+import { Store } from '../../store/store';
+
+export interface PageTileContainerProps {
+	focused: boolean;
+	onClick: React.MouseEventHandler<HTMLElement>;
+	page: PageRef;
+}
+
+@observer
+export class PageTileContainer extends React.Component<PageTileContainerProps> {
+	@MobX.observable public editable: boolean = false;
+	@MobX.observable public inputValue: string = '';
+
+	public constructor(props: PageTileContainerProps) {
+		super(props);
+		this.inputValue = this.inputValue ? this.inputValue : this.props.page.getName();
```

Comment:
What is the difference between this an
```
this.inputValue = this.props.page.getName()
```

Reply:
The `this.inputValue` is a dynamic value that changes with the entered value from the input element. But if the value has not been set, then it should use the current name of the page.